socrates_api/serializers.py

```diff
@@ -652,30 +652,9 @@ class Meta(RethinkSerializer.Meta):
             ('permissions_write', r.row['permissions']['write'], {'multi': True}),
         ]
 
-    def update(self, instance, data):
-        from socrates_api.tasks import firewall_apply
-        ruleset = super(FirewallRuleSetSerializer, self).update(instance, data)
```

Comment:
This is required.

Reply:
was a bit to eager with the scissors, should be fixed now